src/schema/sources.ts

```diff
@@ -1705,18 +1736,14 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
           : undefined,
       };
 
-      if (!isNullOrUndefined(isPrivate) && current.private !== isPrivate) {
-        const approved = await ctx.con
-          .getRepository(SquadPublicRequest)
-          .findOne({
-            where: { sourceId, status: SquadPublicRequestStatus.Approved },
-          });
-
-        if (!approved) {
-          throw new ValidationError(SourceRequestErrorMessage.SQUAD_INELIGIBLE);
+      if (!isNullOrUndefined(isPrivate)) {
```

Comment:
Can we maybe just case `isPrivate` to always boolean like `const isPrivate = !!args.isPrivate` and then we don't need so much nesting of `if` statements in this and `create` mutation.

Reply:
Yeah, why not. Less indents are always welcome.

Reply:
A test failed, I think it is due to the fact that the parameter is not required and that if you send undefined, we are explicitly setting the value, and could be the opposite of what state it needs to be.